mysql-test/suite/galera/t/galera_create_table_as_select.test

```diff
@@ -182,3 +182,29 @@ SHOW TABLES;
 
 --connection node_1
 DROP TABLE t1, t2;
+
+--connection node_1
+CREATE TABLE t2 (f1 INTEGER) ENGINE=InnoDB;
+INSERT INTO t2 VALUES (1),(2),(3),(4),(5);
+
+CREATE TEMPORARY TABLE t1 AS SELECT * FROM t2;
+
+# Add a sleep to delay the completion of the query
+--sleep 100
```

Comment:
This is a wait caused by the MTR's `--sleep` command is not the `SELECT SLEEP(100)` which is done by the connection thread. So, the kill mechanism will fail as the query `SELECT ID FROM INFORMATION_SCHEMA.PROCESSLIST WHERE INFO LIKE '%SLEEP%' LIMIT 1` shall return empty value.